CONTRIBUTING.md

```diff
@@ -25,7 +25,9 @@ contributors.  Typical contribution flow steps are:
 
 - Fork the go-vcloud-director repo into a new repo on GitHub
 - Clone the forked repo locally and set the original go-vcloud-director repo as the upstream repo
-- Make changes in a topic branch and commit
+- Open an Issue in go-vcloud-director describing what you propose to do (unless the change is so trivial that an issue is not needed.)
+- Wait for discussion and possible direction hints in the issue thread.
```

Comment:
Remove the dot at the end of the sentence to be consistent with the rest of the paragraph.